website/docs/r/event_orchestration.html.markdown

```diff
@@ -3,14 +3,14 @@ layout: "pagerduty"
 page_title: "PagerDuty: pagerduty_event_orchestration"
 sidebar_current: "docs-pagerduty-resource-event-orchestration"
 description: |-
-  Creates and manages a Global Event Orchestration in PagerDuty.
+  Creates and manages an Event Orchestration in PagerDuty.
```

Comment:
In the new Orchestrations v2 world, `Global Event Orchestration` refers to a different concept / resource. In this PR, we are renaming this specific metadata resource as `Event Orchestration` to be more in-line with our other public facing docs.